core/backend/src/ElementAspect.ts

```diff
@@ -64,7 +64,9 @@ export class ElementAspect extends Entity {
    * @beta
    */
   protected static onInsert(arg: OnAspectPropsArg): void {
-    arg.iModel.channels.verifyChannel(arg.model);
+    const { props, iModel } = arg;
+    iModel.channels.verifyChannel(arg.model);
+    iModel.locks.checkExclusiveLock(props.element.id, "element", "update");
```

Comment:
the strings are just for the exception string. This should say "insert aspect" rather than "update".

Reply:
I fixed it and error message look like following

`Error inserting ElementAspect [exclusive lock not held on element for insert aspect (id=0x20000000004)], class: BisCore:ExternalSourceAspect`